search-commons/build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins{
     id 'nva.search.service.java-common-conventions'
+    id 'nva.search.service.publish-maven'
```

Comment:
activate the pubish-maven plugin in order to be published in maven central or jitpack